pxf/pxf-service/src/main/resources/pxf-log4j.properties

```diff
@@ -23,7 +23,7 @@ log4j.rootLogger=INFO, ROLLINGFILE
 
 log4j.appender.ROLLINGFILE=org.apache.log4j.RollingFileAppender
 # use the same value directory as set for $PXF_LOGDIR
-log4j.appender.ROLLINGFILE.File=/var/log/pxf/pxf-service.log
+log4j.appender.ROLLINGFILE.File=$${env:$PXF_LOGDIR}/pxf-service.log
```

Comment:
Are two dollar symbols above intentional?

Reply:
Log4j 2.x documentation suggests this way of reading environment var - $${env:USER}. Still testing though.

Reply:
I would assume these properties would be retrieved from Java in which case they would need to be set via -D option when starting Java ?

Reply:
According to docs, Log4J supports both environment and system vars, $${env:USER} and ${sys:logPath} respectively.